msp/compliance/orgs_and_cloud_accounts_report/orgs_and_cloud_accounts_report.pt

```diff
@@ -0,0 +1,280 @@
+name "Orgs and Clouds Vendor Accounts Report Policy"
+rs_pt_ver 20180301
+type "policy"
+short_description "This policy generates a list of cross organization Cloud Vendor Accounts connected to Flexera Optima based on
+the bill connection settings for Azure and Google, as well as full list of AWS accounts under the payer account connected for each Flexera Organization.\n
+_Note 1: 'enterprise_manager' user role is required for each organization._
+_Note 2: If excluding orgs is needed use only either 'Excluded Organizations' or 'Exclucded organizations IDs' parameter, not both._
```

Comment:
Keep the short description short.  The _Notes are not needed here since they are already mentioned in the readme.  Just make sure the readme has all the necessary details.

Reply:
Done.